config/configs/geolix.exs

```diff
@@ -0,0 +1,12 @@
+use Mix.Config
+
+config :geolix,
+  databases: [
+    %{
+      id: :country,
+      adapter: Geolix.Adapter.MMDB2,
+      init: {MaxMindInitializer, :init}
+    }
+  ],
+  banned_countries: System.get_env("BANNED_COUNTRIES"),
+  database_location: System.get_env("DATABASE_LOCATION")
```

Comment:
Just a suggestion.. maybe we should use something like `MMDB_LOCATION` to make it so devs don't think it's location for Postgres

Reply:
good call